src/scenes/game.js

```diff
@@ -1,17 +1,85 @@
 //Creates phaser class for the scene
 class GameScreen extends Phaser.Scene {
 
-	constructor() {
+	constructor(data) {
 		//the key that references this class, which is used by other functions
 		super('GameScreen')
 	}
 
+	//Basically, this part gets the data from the character select screen
+	init (data)
+	{
+		//This is the sprite object that is passed in
+	    this.choiceLeft = data.playerLeftChoice;
+	    this.choiceRight = data.playerRightChoice;
+	    //This is the array for which frames to start and stop animation. [0] and [1] is for running, [2] and [3] is for attacking.
+	    this.leftFrameArray = data.playerLeftFrames;
+	    this.rightFrameArray = data.playerRightFrames;
+	    //This is the array representing the stats of a player. Strength is [0] and speed is [1]
+		this.leftStrength = data.playerLeftStats[0];
+		this.leftSpeed = data.playerLeftStats[1];
+		this.rightStrength = data.playerRightStats[0];
+		this.rightSpeed= data.playerRightStats[1];
+	}
+	
 	preload() {
-	  	
+
 	}
 
 	create() {
-	 	
+		//THIS WHOLE THING IS TESTING
+		console.log(this.choiceLeft);
+		console.log(this.leftStrength);
+		console.log(this.leftSpeed);
+
+		//Because we get a spritesheet object from the character select screen, we need to create the sprite based off of that object.
+		let jousterLeft = this.physics.add.sprite(this.choiceLeft.x, this.choiceLeft.y, this.choiceLeft.texture.key, this.choiceLeft.frame.name);
+		jousterLeft.flipX = true;
```

Comment:
Turns out we can mirror a sprite, so thankfully we don't need separate animation sheets for going left and right.